test/plugins/features/movies/controller.test.js

```diff
@@ -1,19 +1,43 @@
 'use strict';
 
+const Knex = require('../../../../lib/libraries/knex');
+
 const Controller = require('../../../../lib/plugins/features/movies/controller');
 
 describe('movie controller', () => {
 
-  describe('create', () => {
+  const payload = { name: 'test', release_year: 2000 };
+  let movie = {};
 
-    it('creates a movie', async () => {
-      const payload = { name: 'foo' };
+  before(async () => {
+    await Knex.truncate('movies');
+    movie = await Controller.create(payload);
+  });
```

Comment:
I'd recommend using a `beforeEach` here and only truncating the table. I'd recommend only using the `beforeEach` blocks for test set up. I know we talked about using `create` for insertion as set up for the fetches but since we're also testing `create` as well I think it makes more sense for us to call `create` in `it creates a movie`. Then we can call `create` in the `beforeEach` for the `describe` block for the fetches. What are your thoughts on this?